rpcs3/Emu/RSX/Common/TextureUtils.cpp

```diff
@@ -53,6 +53,38 @@ struct copy_unmodified_block_swizzled
 	}
 };
 
+struct copy_unmodified_block_vtc
+{
+	template<typename T, typename U>
+	static void copy_mipmap_level(gsl::span<T> dst, gsl::span<const U> src, u16 width_in_block, u16 row_count, u16 depth, u32 dst_pitch_in_block, u32 src_pitch_in_block)
+	{
+
```

Comment:
Unnecessary new line at beginning of function. The newline with the brace already does the separation for you